### ISSUE TITLE
pypi package

### ISSUE THREAD

**Issue by AbdealiJK** [Yaafe/Yaafe#20]
It would be good to have a pypi package for yaafe for easier installation and also so that it does python dependency handling

**Comment by AbdealiJK** [comment]
I'm not sure how this would be implemented.
`numpy` has custom distutils Commands that compile things for it. But this would be an alternative for the `cmake` method. See: https://github.com/numpy/numpy/tree/master/numpy/distutils

So the question would be: Do you want to maintain 2 `make` systems (CMake and the setuptools one) or do you want to replace one of them ?
I guess a simple method would be to just do `subprocess.call` with the cmake command ... and throw errors are appropriate. we

**Comment by AbdealiJK** [comment]
I found `dlib` has a cmake installer too, and also a pypi package.
Their setup.py calls cmake using Popen

**Comment by jayvdb** [comment]
Does Yaafe have Windows support, or plans for support? Does that need to be a consideration for this?

**Comment by thomasfillon** [comment]
Yes Yaafe can compile on Windows thanks to @Crococode  and @gholzmann great contributions.

see https://github.com/Yaafe/Yaafe/blob/master/INSTALL_WINDOWS

**Comment by jayvdb** [comment]
Then I would recommend working towards building wheels using cmake, that can be uploaded to pypi, so that users do not need to build all the dependencies on Windows.  The wheel contains all the dlls, and the python code.  Also a wheel can be built entirely by hand, or convoluted steps, if necessary.  A wheel does not need a `setup.py`, and neither setuptools nor distlib are mandatory, etc.

Once the wheel is built, and uploaded to pypi, the user has no trouble installing it using pip or setuptools.

**Comment by StanSilas** [comment]
Any update on this feature? I want to install yaafe as `pip install yaafe` or something along those lines.
